README.md

```diff
@@ -57,6 +57,17 @@ Download and install [librdkafka](https://github.com/edenhill/librdkafka/). Run
 
 Installing PHP extension:
 ----------
+
+#### With Homebrew (For Mac users)
+
+```bash
+$ brew install php56-kafka
+```
+
+Note: Install `php53-kafka`, `php54-kafka`, `php55-kafka`, `php56-kafka` or `php70-kafka` depends on PHP version on your machine.
```

Comment:
`php70-kafka` doesn't exist (yet): The current extension is incompatible with PHP7